PeerioTechnologies/peerio-desktop
# fix(files): make file and folder icons clickable

#### Relevant info and issue/PR links 
 
https://app.clubhouse.io/peerio/story/1185/icons-in-files-are-not-clickable

#### Testing instructions  

In Files, click on a file icon — it should open the download dialog. Click on a folder icon — it should open this folder.

In a chat, click +, Share from my files, and try clicking on the file icon. It should select the file. Clicking on a folder icon should open the folder.

Make sure nothing else that has a file list get broken.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
